src/components/navbar/navbar.component.tsx

```diff
@@ -13,7 +13,7 @@ import {
   HeaderGlobalAction
 } from "carbon-components-react/es/components/UIShell";
 import { LoggedInUser, UserSession } from "../../types";
-import styles from "./UserAvatar_styles.css";
+import styles from "./navbar.component.scss";
```

Comment:
How is styles used?

Because we use CSS modules and right now I don't see how the style is applied to the right position.

Reply:
@FlorianRappl,
In this commit, I made the changes related to the file name.
If you look at the previous commit, there the style changes were applied.
![image](https://user-images.githubusercontent.com/61249902/113148740-d9423a80-924f-11eb-9a54-053099f2741f.png)

Reply:
Not sure how this answers my question. The import was not used in this file beforehand. So how is it relevant / used?